jlm/hls/backend/rhls2firrtl/RhlsToFirrtlConverter.cpp

```diff
@@ -321,7 +321,7 @@ RhlsToFirrtlConverter::MlirGenSimpleNode(const jlm::rvsdg::simple_node * node)
       int bits = JlmSize(pointeeType);
       if (dynamic_cast<const jlm::rvsdg::bittype *>(pointeeType))
       {
-        ;
+        pointeeType = nullptr;
```

Comment:
I do not understand why you set `pointeeType` to `nullptr` here. This does not make sense to me at all. The only place where `pointeeType` is used is in the `JlmSize()` method, and this method does not really handle `nullptr` gracefully.